### PR TITLE
feat(plaza): show current sign up bonus on plaza for new players

### PR DESCRIPTION
**Pull request by kcper**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?
image
image

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?
#407 
### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Show promotion info when wallet has 0 characters.

### CODE REVIEW

File: frontend/src/views/Plaza.vue [hunk 1]
```diff
@@ -2,6 +2,19 @@
   <div class="body main-font">
 
     <div v-if="ownCharacters.length === 0" class="blank-slate">
+      <div class="current-promotion">
+        <div class="tob-bg-img promotion-decoration">
+          <img class="vertical-decoration bottom" src="../assets/border-element.png">
+        </div>
+        <strong>SIGN UP BONUS</strong>
```
Comment (seiyria) [inline frontend/src/views/Plaza.vue:9]: please don't do uppercase text in code - add a class that does text-transform: uppercase

**Review by seiyria — CHANGES_REQUESTED** [review]
I like this, but can we get a little more space between the images and the text? maybe like 10 pixels on top and bottom

**Comment by kcper** [comment]
Sure :)
image

**Comment by seiyria** [comment]
way better

**Review by seiyria — APPROVED** [review]

**Pull request merged**